### PR TITLE
Prac 07 feedback

### PR DESCRIPTION
**Pull request by Jasper-afk**
Could you please review my code's efficiency and if my variable and function naming is clear and concise? @Moggie269

### CODE REVIEW

File: prac_07/project.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+"""
+CP1404 Practical 7 - Project class file.
+"""
+
+class Project:
+    """Represent what will be in a Project object."""
+
+    def __init__(self, name, start_date, priority, cost_estimate, completion_percentage):
```
Comment (Moggie269) [inline prac_07/project.py:8]: Did you consider having a Docstring here?

File: prac_07/project.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.cost_estimate = cost_estimate
+        self.completion_percentage = completion_percentage
+
+    def __repr__(self):
```
Comment (Moggie269) [inline prac_07/project.py:16]: Did you consider having a Docstring here?

**Review by Moggie269 — COMMENTED** [review]
Overall, a great work

File: prac_07/project.py (hunk 1 above)
Comment (Jasper-afk) [reply prac_07/project.py:8]: Oh, I totally forgot. I'll fix it now.

**Comment by Jasper-afk** [comment]
I appreciate your review on my work and thank you for pointing out my missing docstrings.

**Pull request merged**